src/core/engine/processors/kitchen_simulator.py

```diff
@@ -9,5 +12,49 @@ def __init__(self, app_processor_config, app_context=None):
             app_processor_config=app_processor_config, app_context=app_context
         )
 
+        ioc = get_ioc_instance()
+        self.order_controller = ioc.get_instance("order_controller")
+        self.order_status_history_controller = ioc.get_instance(
+            "order_status_history_controller"
+        )
+        self.chef_controller = ioc.get_instance("chef_controller")
+        self.order_manager = OrderManager()
+
     def process(self, delta_time):
         print("Waiting orders")
+        self.assign_orders_to_available_chefs()
+
+    def assign_orders_to_available_chefs(self):
+
+        orders_to_process = self.order_controller.get_orders_to_process()
+        orders_validation_map = self.order_controller.get_validated_orders_map(
+            orders_to_process
+        )
+        available_chef_ids = self.chef_controller.get_available_chefs()
+
+        if available_chef_ids:
+
+            order_in_turn_id = self.order_manager.get_queue_from_status("new_order")
```

Comment:
Could you create an enum for the naming of the different queues?

Reply:
done.